lib/modules/datasource/custom/readme.md

```diff
@@ -273,3 +310,47 @@ And the following regex manager:
   ]
 }
 ```
+
+### nginx directory listing
+
+Sometimes all you have is a directory with files, and a HTTP server that can generate directory listings.
+
+Let's use nginx itself as an example:
+
+```json
+{
+  "customDatasources": {
+    "nginx": {
+      "defaultRegistryUrlTemplate": "http://nginx.org/download",
```

Comment:
Is nginx still using HTTP (without the S)? I'd prefer if we could show a HTTPS example, if at all possible.